plane_default.yaml

```
@@ -0,0 +1,21 @@
+DOWNLOADS:
+  URL: "https://b2share.fz-juelich.de/api/files/a423a576-220d-47b0-9e0c-b5b32d45fc59"
+  URL_BACKUP: "https://zenodo.org/records/10390573/files"
```

```suggestion
  URL:
  - "https://b2share.fz-juelich.de/api/files/a423a576-220d-47b0-9e0c-b5b32d45fc59"
  - "https://zenodo.org/records/10390573/files"
```

This probably causes some other changes as it is a list now